src/CalcManager/CEngine/scioper.cpp

```diff
@@ -112,8 +112,17 @@ CalcEngine::Rational CCalcEngine::DoOperation(int operation, CalcEngine::Rationa
             }
             else
             {
-                iFinalSign = iNumeratorSign;
-                result %= temp;
+                if (m_fIntegerMode)
+                {
+                    // Programmer mode
+                    iFinalSign = iNumeratorSign;
+                    result %= temp;
+                }
+                else
+                {
+                    iFinalSign = iDenominatorSign;
```

Comment:
There was some discussions about having 2 buttons instead of 1 (`mod` and `rem`), even if not used, I prefer to still indicate that the sign is not calculated the same way.

Reply:
Understood - given that it isn't currently used, however, I'd recommend commenting it out and adding a short comment noting that it's unused but represents the logic if arithmetic mod is added to programmer mode.